.github/workflows/push.yml

```diff
@@ -0,0 +1,137 @@
+name: 'GitHub Actions'
+'on':
+  - push
+concurrency: 'ci-${{ github.ref }}'
+env:
+  APP_IMAGE: keboola-component
+  KBC_DEVELOPERPORTAL_VENDOR: keboola
+  KBC_DEVELOPERPORTAL_APP: keboola.wr-synapse
+  KBC_DEVELOPERPORTAL_USERNAME: keboola+wr_synapse_github_actions
+  KBC_DEVELOPERPORTAL_PASSWORD: '${{ secrets.KBC_DEVELOPERPORTAL_PASSWORD }}'
+
+  AZURE_RESOURCE_GROUP: synapse-writer-ci
+  AZURE_SERVICE_PRINCIPAL: '${{ secrets.AZURE_SERVICE_PRINCIPAL }}'
+  AZURE_SERVICE_PRINCIPAL_PASSWORD: '${{ secrets.AZURE_SERVICE_PRINCIPAL_PASSWORD }}'
+  AZURE_SERVICE_PRINCIPAL_TENANT: '${{ secrets.AZURE_SERVICE_PRINCIPAL_TENANT }}'
+  SERVICE_PRINCIPAL_NAME: ci-db-writer-synapse
+  SYNAPSE_PORT: 1433
+  SYNAPSE_SERVER_NAME: synapse-writer-ci-server
+
+  DOCKERHUB_USER: '${{ secrets.DOCKERHUB_USER }}'
+  DOCKERHUB_TOKEN: '${{ secrets.DOCKERHUB_TOKEN }}'
+  KBC_STORAGE_TOKEN: '${{ secrets.KBC_STORAGE_TOKEN }}'
+
+  KBC_URL: https://connection.eu-central-1.keboola.com
+  KBC_PROJECT: https://connection.eu-central-1.keboola.com/admin/projects/2835/storage-explorer
+  KBC_RUNID: test-run-dev
+  STORAGE_API_TOKEN: '${{ secrets.STORAGE_API_TOKEN }}'
+
+  KBC_TEST_PROJECT_URL: ''
+  KBC_TEST_PROJECT_CONFIGS: ''
```

Comment:
když si smazal tests-kbc tak to tu je zbytečný

Reply:
smazáno 👍